schemes.yaml

```diff
@@ -962,6 +962,35 @@ schemes:
       cyan:    '0x9aedfe'
       white:   '0xf1f1f0'
 
+  seashells: &seashells
+    primary:
+      background: '0x61923'
```

Comment:
This background value gave me an error - `0x061923` worked however.

Reply:
Yeah, should be 0x061923, as it should be 6 digital Hex number :)